syntax = "proto3";

package chatbot;

// Messaggio di input dal client
message UserMessage {
  string message = 1;
}

// Messaggio di risposta dal server
message BotResponse {
  string intent = 1;
  string response = 2;
}

// Servizio gRPC
service ChatbotService {
  rpc Query (UserMessage) returns (BotResponse);
}
